### ISSUE TITLE
seed를 동일하게 하여 학습을 진행했음에도 불구하고 loss log가 동일하지 않은 issue

### ISSUE THREAD

**Issue by sawo0150** [LISTatSNU/FastMRI_challenge#271]
성심당 팀입니다.

현재 vessl 서버와 local 컴퓨터 총 2곳에서 모델을 학습하고 있습니다.
![localTraining](https://github.com/user-attachments/assets/1ec6ad96-50a7-4fe6-ba5e-6f50d3f8c02b)
![vesslserver_training](https://github.com/user-attachments/assets/afa29401-afc6-4591-8630-8bcbf4279377)

두 곳의 학습 로그를 캡처한 것인데, 학습 코드에서 분명 seed를 고정하였지만, 처음에는 loss가 일치하다가 뒤로 갈수록 조금씩 loss 값에서 차이가 발생하게 됩니다. 이러한 차이는 epoch 1이 지났을 때는 꽤 커지게 되어서 val_loss_log 파일의 값에서 차이가 나게 됩니다.

차이의 원인으로써 의심되는 것은 사용하는 GPU 모델이 다음에 따라 NVIDIA driver와 Cuda 버전이 차이 나는 것으로 인해 시드가 동일하지만 loss 값이 조금씩 차이 나게 되는 것 같습니다.

평가에서는 이러한 차이에 대해서 직접 검증할 수 있으면 괜찮은가요? 아니면 이러한 loss 값의 차이가 안 나오도록 바꾸는 방법이 있나요?

감사합니다.

**Comment by yxxshin** [comment]
@sawo0150 안녕하세요,

말씀해 주신 대로 GPU 환경에 따라 같은 seed 값에서도 val loss가 서로 다를 수 있음을 인지하고 있습니다.
조교 측에서는 두 val loss 값이 '유사'한 지 확인할 것이며, 만약 차이가 유의미하게 난다고 판단이 되면 추가 검증, 직접 시연 등을 요청할 수 있습니다.

감사합니다.

**Comment by sawo0150** [comment]
넵 감사합니다
